utils/src/newtype.rs

```diff
@@ -65,9 +65,10 @@ mod tests {
         }
     }
 
-    newtype!(
-        #[derive(Clone, Debug)]
-        struct NewInt(OldInt));
+    newtype! {
+    #[derive(Clone, Debug)]
+    struct NewInt(OldInt);
+    }
```

Comment:
I suppose the indentation here should be fixed manually.

Reply:
Or we can try to use some of the external crates. E.g. [derive-newtype](https://crates.io/crates/derive-newtype) provides a derive macro which does essentially the same thing but rustfmt works on it.

Reply:
Was not familiar with that crate @iljakuklic . For now it seems like less work to just fix the indentation manually, so I've done that.